company/goals/index.md

```diff
@@ -0,0 +1,72 @@
+# Goals
+
+Our company goals are public:
+
+- [New *continuously updated* goals starting as of FY20-Q3 (upcoming, not publicly visible yet, will be moved to handbook soon)](https://docs.google.com/document/d/1Z6avFUmnGW-ZC6zrktqqQd_g5mhBE96C_mTwXwFc1H4/edit#)
+- Historical snapshots:
+  - [FY20-Q2 (active)](2020_q2.md)
+  - [FY20-Q1](2020_q1.md)
+  - [CY19-Q4](2019_q4.md)
+  - [CY19-Q3](2019_q3.md)
+
+## Philosophy
+
+We believe:
+
+- "Plans are useless, but planning is essential."
+- "No plan survives first contact with the customer."
+- "A plan is only useful for determining the immediate next thing to do. Beyond that, you must be ready to abandon the plan in the face of new evidence."
+- Discarding a goal that turns out to be wrong is better than following through on the wrong goal. People know this intellectually but disregard it in practice.
+- Each problem we're solving has a different (and often unknown/unknowable) timeline and rate of progress.
+- Only people with skin in the game can set useful goals.
+
+## Goals are continuous, not quarterly
+
+1. Goals are continuously updated, and each goal specifies its own time period. They are not (for example) quarterly.
+   - No single update/review cadence (such as quarterly) makes sense for all goals.
+1. Change your goal immediately if/when you discover it's the wrong goal.
+   - Reflect on why you needed to change the goal (so you can learn), but don't keep working toward the wrong goal.
+
+## Choosing goals
+
+A goal's format is:
+
+> **Goal title** \
+> Description of the goal and how to evaluate whether we met the goal (with a link to an automatically updated metrics dashboard, if appropriate).
+
+1. Each goal has a single person (not multiple people) who's ultimately responsible for it.
+   - Many people can be working toward a goal, but there must be one person who's ultimately responsible.
+1. No person can have more than 7 goals at once. (This number is arbitrary, but it feels right.) Fewer is better.
+1. Pick goals where failure would be painful.
+1. Pick goals where you can influence the outcome. Avoid using a lagging indicator as a goal.
+1. For a new project, it can be hard to define the right goal and measurement. Do your best. Then refine the goal as you learn more.
+1. Set goals to be ambitious but achievable, so that they convey your aims and what you could achieve if things go well.
+   - In (rough) numeric terms, if you achieve less than 70% of your goal, it was probably not achievable. If you regularly achieve 100%+ of your goals, they are probably not ambitious enough.
```

Comment:
Since these goals are continuous, I would expect that we're working towards 100% on each goal (presuming this goal is still important and has not been invalidated). 

Another concern/question - it feels likely that we will start to reach 70% on a goal, declare it good enough and move on to a new goal. Perhaps that is ok, but would indicate that goals are too hard, when in fact we just chose "good enough" and the true goal was actually appropriate.

Reply:
Yes, good point. I removed this because the 70% is no longer relevant.